queries/plugins/text_to_num.py

```diff
@@ -0,0 +1,573 @@
+
+TöluðTalaKvaðrilljarðar →
+    TöluðTalaUndirKvaðrilljarði? TöluðTalaKvaðrilljarður
+
+# KVAÐRILLJÓNIR
+
+TöluðTalaUndirKvaðrilljarði →
+    TöluðTalaKvaðrilljónir TöluðTalaUndirKvaðrilljón?
+    | TöluðTalaKvaðrilljónir TöluðTalaOgUndirKvaðrilljón
+    | TöluðTalaUndirKvaðrilljón
+
+TöluðTalaOgUndirKvaðrilljarði →
+    "og" TöluðTalaKvaðrilljónir
+    | TöluðTalaOgUndirKvaðrilljón
+
+TöluðTalaKvaðrilljónir →
+    TöluðTalaUndirKvaðrilljón? TöluðTalaKvaðrilljón
+
+# TRILLJARÐAR
+
+TöluðTalaUndirKvaðrilljón →
+    TöluðTalaTrilljarðar TöluðTalaUndirTrilljarði?
+    | TöluðTalaTrilljarðar TöluðTalaOgUndirTrilljarði
+    | TöluðTalaUndirTrilljarði
+
+TöluðTalaOgUndirKvaðrilljón →
+    "og" TöluðTalaTrilljarðar
+    | TöluðTalaOgUndirTrilljarði
+
+TöluðTalaTrilljarðar →
+    TöluðTalaUndirTrilljarði? TöluðTalaTrilljarður
+
+# TRILLJÓNIR
+
+TöluðTalaUndirTrilljarði →
+    TöluðTalaTrilljónir TöluðTalaUndirTrilljón?
+    | TöluðTalaTrilljónir TöluðTalaOgUndirTrilljón
+    | TöluðTalaUndirTrilljón
+
+TöluðTalaOgUndirTrilljarði →
+    "og" TöluðTalaTrilljónir
+    | TöluðTalaOgUndirTrilljón
+
+TöluðTalaTrilljónir →
+    TöluðTalaUndirTrilljón? TöluðTalaTrilljón
+    | TöluðTalaUndirMilljarði? TöluðTalaMilljarður TöluðTalaMilljarður
+
+# BILLJARÐAR
+
+TöluðTalaUndirTrilljón →
+    TöluðTalaBilljarðar TöluðTalaUndirBilljarði?
+    | TöluðTalaBilljarðar TöluðTalaOgUndirBilljarði
+    | TöluðTalaUndirBilljarði
+
+TöluðTalaOgUndirTrilljón →
+    "og" TöluðTalaBilljarðar
+    | TöluðTalaOgUndirBilljarði
+
+TöluðTalaBilljarðar →
+    TöluðTalaUndirBilljarði? TöluðTalaBilljarður
+    | TöluðTalaUndirMilljón? TöluðTalaMilljón TöluðTalaMilljarður
+
+# BILLJÓNIR
+
+TöluðTalaUndirBilljarði →
+    TöluðTalaBilljónir TöluðTalaUndirBilljón?
+    | TöluðTalaBilljónir TöluðTalaOgUndirBilljón
+    | TöluðTalaUndirBilljón
+
+TöluðTalaOgUndirBilljarði →
+    "og" TöluðTalaBilljónir
+    | TöluðTalaOgUndirBilljón
+
+TöluðTalaBilljónir →
+    TöluðTalaUndirBilljón? TöluðTalaBilljón
+    | TöluðTalaUndirMilljón? TöluðTalaMilljón TöluðTalaMilljón
+
+# MILLJARÐAR
+
+TöluðTalaUndirBilljón →
+    TöluðTalaMilljarðar TöluðTalaUndirMilljarði?
+    | TöluðTalaMilljarðar TöluðTalaOgUndirMilljarði
+    | TöluðTalaUndirMilljarði
+
+TöluðTalaOgUndirBilljón →
+    "og" TöluðTalaMilljarðar
+    | TöluðTalaOgUndirMilljarði
+
+TöluðTalaMilljarðar →
+    TöluðTalaUndirMilljarði? TöluðTalaMilljarður
+
+# MILLJÓNIR
+
+TöluðTalaUndirMilljarði →
+    TöluðTalaMilljónir TöluðTalaUndirMilljón?
+    | TöluðTalaMilljónir TöluðTalaOgUndirMilljón
+    | TöluðTalaUndirMilljón
+
+TöluðTalaOgUndirMilljarði →
+    "og" TöluðTalaMilljónir
+    | TöluðTalaOgUndirMilljón
+
+TöluðTalaMilljónir →
+    TöluðTalaUndirMilljón? TöluðTalaMilljón
+
+# ÞÚSUND
+
+TöluðTalaUndirMilljón →
+    TöluðTalaÞúsundir TöluðTalaUndirÞúsund?
+    | TöluðTalaÞúsundir TöluðTalaOgUndirÞúsund
+    | TöluðTala10Til19Hundruð TöluðTalaUndirHundrað?
+    | TöluðTala10Til19Hundruð TöluðTalaOgUndirHundrað?
```

Comment:
These two productions are mutually ambiguous. Refine the grammar so that only one production can match a token string that doesn't end with "TöluðTalaUndirHundrað" or "TöluðTalaOgUndirHundrað".